libgalois/include/katana/Properties.h

```diff
@@ -505,5 +505,98 @@ struct StructProperty
   }
 };
 
+
+template <typename T>
+class VectorRef {
```

Comment:
This should be documented. IT's unclear what it's for and why it's so insanely mutable.

Reply:
I added documentation and removed setters.